lib/utilshare_web/controllers/household_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule UtilshareWeb.HouseholdController do
+  use UtilshareWeb, :controller
+
+  alias Utilshare.Accounts
+  alias Utilshare.Accounts.Household
+
+  action_fallback UtilshareWeb.FallbackController
+
+
+  def create(conn, %{"household" => household_params}) do
+    IO.puts "Auth"
+    IO.inspect conn.assigns.auth
+    with {:ok, %Household{} = household} <- Accounts.create_household(household_params) do
```

Comment:
What happens if this fails?

Reply:
the fallback controller handles catching insertion errors.